isis/tests/cnetcheckFunctionalTests.cpp

```diff
@@ -76,27 +72,24 @@ TEST_F(ThreeImageNetwork, FunctionalTestCnetcheckNoPoints) {
   // remove all controlpoints from test network
   QList<ControlPoint*> cpoints = network->GetPoints();
 
-  for (size_t i = 0; i<cpoints.size(); i++) {
+  for (int i = 0; i<cpoints.size(); i++) {
```

Comment:
Why int? In this case, it will probably not matter but seems like a weird change.

Reply:
It raised a warning because QT containers return ints for size instead of unsigned ints.